src/controllers/MealSessionController.js

```diff
@@ -181,19 +182,22 @@ async function verifyStudentInMealSession(request, response) {
     return response.status(400).json(generateFormattedError(error));
   }
 
+  if (!StudentService.read(studentId)) {
+    return response.status(404).json({
+      error: {
+        message: 'Unable to find requested student!',
```

Comment:
Padronizar mensagens:
```suggestion
        message: 'Student not found!',
```